lib/main_common.pm

```diff
@@ -88,6 +89,11 @@ sub logcurrentenv {
     }
 }
 
+sub is_tumbleweed {
+    return 1 if check_var('VERSION', 'Tumbleweed');
+    return check_var('DISTRI', 'opensuse') && get_var('VERSION') =~ /^Staging:[A-Z]/;
```

Comment:
Good, then move the commit comment into the file. The commit message should describe the need for change, the in file comments can describe what the code should do, isn't it?